dashboard/src/routes/dashboard/index.svelte

```diff
@@ -0,0 +1,152 @@
+<script lang="ts">
+	import { createDevice, getDevices } from '$lib';
+	import type { Device } from '$lib/types';
+	import {
+		authenticationData,
+		Icon,
+		ListManager,
+		NotificationManager,
+		type ActionButton,
+		type TableData
+	} from 'boxfish-studio--is-ui-components';
+	import { onMount } from 'svelte';
+	import { Container, Row } from 'sveltestrap';
+	import { DeviceDetails, ProgressBar } from '../../components';
+	import { progress } from '$lib/store';
+
+	enum State {
+		ListDevices = 'listDevices',
+		DeviceDetails = 'deviceDetails'
+	}
+
+	let devices = [];
```

Comment:
missing type

Reply:
why do we need this variable here? cant it be a local constant in `loadDevices`

Reply:
we need `devices`variable in `onSearch` function